drake/solvers/CMakeLists.txt

```diff
@@ -32,28 +32,23 @@ if(MATLAB_FOUND)
   target_link_libraries(lqrmex drakeUtil)
 endif()
 
+set(optimization_files)
+if (snopt_c_FOUND)
+  list(APPEND optimization_files SNOPTWrapper.cpp)
+else()
+  list(APPEND optimization_files NoSNOPT.cpp)
+endif(snopt_c_FOUND)
+include_directories(${PROJECT_SOURCE_DIR}/core)
+add_library_with_exports(LIB_NAME drakeOptimization SOURCE_FILES ${optimization_files})
+set_target_properties(drakeOptimization PROPERTIES COMPILE_FLAGS -fPIC)
+if (snopt_c_FOUND)
+  pods_use_pkg_config_packages(drakeOptimization snopt_c)
+endif(snopt_c_FOUND)
+
 if(snopt_c_FOUND AND MATLAB_FOUND)
   add_mex(NonlinearProgramSnoptmex NonlinearProgramSnoptmex.cpp)
   pods_use_pkg_config_packages(NonlinearProgramSnoptmex snopt_c)
   set_target_properties(NonlinearProgramSnoptmex PROPERTIES COMPILE_FLAGS -fPIC)
-
-  # link against matlab's copy of the gfortran library, if possible
-  find_library(GFORTRAN_LIBRARY NAMES gfortran libgfortran
```

Comment:
Ah great. Does that mean we should get rid of gfortran as an ubuntu prereq in the snopt install_prereqs.sh?